src/js/_enqueues/wp/theme.js

```diff
@@ -118,9 +119,11 @@ themes.view.Appearance = wp.Backbone.View.extend({
 		// Render and append after screen title.
 		view.render();
 		this.searchContainer
+			.find('.search-box')
 			.append( $.parseHTML( '<label for="wp-filter-search-input">' + l10n.search + '</label>' ) )
```

Comment:
```
.find( '.search-box' )
.append( $.parseHTML( `<label for="wp-filter-search-input">${ 10n.search }</label>` ) )
```

Reply:
The `'.search-box'` selector could have spaces around it (as noted above), but these lines need to [remain indented](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#chained-method-calls).